src/pageActionButton.test.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import { findComponentsWithType } from 'meetup-web-mocks/lib/testUtils';
+import PageActionButton, {
+	PAGE_ACTION_BUTTON_CLASS
+} from './PageActionButton';
+
+const label = 'hello world';
+const shape = 'search';
+const spyable = {
+	onClick: () => {}
+};
+
+let pageActionButton,
+	pageActionButtonEl;
+
+describe('PageActionButton', function() {
+	beforeEach(() => {
+		pageActionButton = TestUtils.renderIntoDocument(
+			<PageActionButton
+				icon={shape}
+				label={label}
+				onClick={spyable.onClick}
+			/>
+		);
+		pageActionButtonEl = ReactDOM.findDOMNode(pageActionButton);
+	});
+
+	afterEach(() => {
+		pageActionButton = null;
+		pageActionButtonEl = null;
+	});
+
+	it('exists', function() {
+		expect(pageActionButtonEl).not.toBeNull();
+	});
+
+	it(`should render a 'button' as the root element with '${PAGE_ACTION_BUTTON_CLASS}'`, () => {
+		expect(pageActionButtonEl.nodeName).toBe('BUTTON');
+		expect(pageActionButtonEl.classList).toContain(PAGE_ACTION_BUTTON_CLASS);
+	});
+
+	it('should render a `Button` component with `onClick` prop', () => {
+		const button = findComponentsWithType(pageActionButton, 'Button');
```

Comment:
more info - @nlaz brought up the `findRenderedComponentsWithType` method with me last week and wondered why we've been using the `findComponentsWithType` function that I wrote ages ago instead. `findComponentsWithType` does a similar search, although it takes a string rather than a reference to a class, which is helpful when you don't have a convenient direct reference to the class you're searching for, but it otherwise a little bit roundabout, and I'm not sure it even works with pure functional components.

Going forward, `TestUtils.findRenderedComponentsWithType` will almost always be the preferred method to do this kind of check.